app/src/components/lanes/itemLane/itemLane.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+import {
+  Box,
+  Flex,
+  Heading,
+  Link,
+  SimpleGrid,
+  Spacer,
+} from "@nypl/design-system-react-components";
+import { useNumColumns } from "../../../hooks/useNumColumns";
+import useBreakpoints from "../../../hooks/useBreakpoints";
+import ItemCard from "../../cards/itemCard";
+import { ItemCardModel } from "../../../models/itemCard";
+import React from "react";
+import { titleToDCParam, totalNumPages } from "../../../utils/utils";
+import { DC_URL } from "@/src/config/constants";
+
+export const ItemLane = ({ data }: any) => {
+  const numColumns = useNumColumns();
+  const { isLargerThanLargeTablet } = useBreakpoints();
+  const divisionName = data.name;
+  return (
+    <>
+      <Box>
+        <Flex alignItems="baseline">
+          <Heading level="h2" size="heading3">
+            {`Items in the ${divisionName}`}
+          </Heading>
+          <Spacer />
+          <Link
+            id={`row-see-more-items-${divisionName}`}
+            type="standalone"
+            href={`${DC_URL}/search/index?filters[divisionFullname_mtxt_s][]=${titleToDCParam(
```

Comment:
That's an interesting looking URL. Out of curiosity and not blockers: would you happen to know why it's `+` instead of the browser' `%20` for spaces? Or why an URL encoded title can't be passed?

Reply:
given that the filter is `divisionFullname_mtxt_s`, I'm assuming it's a solr thing. perhaps solr use use both `AND` and `+`, but I can ask Kris if we need to know. I'm just matching what I see in the legacy dc lol